utils/guidelines_xslt/odd2html/specs/macroGroupSpecs.xsl

```diff
@@ -54,7 +54,7 @@
     
     <xd:doc>
         <xd:desc>
-            <xd:p>Creates a section for a single macroSpec[@type='pe']</xd:p>
+            <xd:p>Creates a section for a single macroSpec</xd:p>
```

Comment:
There is a typo in line 40 above. It should read:

`<xd:p>Creates a section with all macroSpecs</xd:p>`

Can you please fix it, too?